src/ddmd/mars.d

```diff
@@ -245,6 +245,29 @@ extern (C++) void genCmain(Scope* sc)
     rootHasMain = sc._module;
 }
 
+private void semantic3OnDependencies(Module m, StringTable *importArray)
```

Comment:
This function does not look like it belongs in `mars.d`, which is not common between the three frontends.  Most likely it belongs in `dmodule.d`.